### DISCUSSION TITLE
Meeting meets twice a month

### DISCUSSION THREAD

**Discussion by WoodieFL** [code4recovery/12-step-meeting-list#761] [Q&A] [answered]
Is there any way to show a meeting meets on the 2nd and 4th Thursday, other than putting that in the notes?

**Comment by kiyote33** [comment, marked as answer]
Hi @WoodieFL, You've already hit on our suggested answer to this question. Our plugin was designed to list reoccurring weekly meetings, and other than putting something in the Notes about your by-weekly meet there isn't anything else we can suggest.

**Comment by joshreisner** [comment]
Hi @WoodieFL Meeting Guide app support also recommends listing these meetings on a separate page rather than using the main meeting directory, if it's a concern that users might not tap on the meeting details and read the notes.

**Comment by WoodieFL** [comment]
Thanks, Wayne. No problem at all.

**Comment by WoodieFL** [comment]
Thanks for your quick response. No problem.
